config/tests/fixtures/absolute_paths.toml

```diff
@@ -11,4 +11,5 @@ out_file = "/telemetry/file.json"
 
 [genesis]
 file = "/oh/my/genesis.json"
-private_key = { algorithm = "ed25519", payload = "8f4c15e5d664da3f13778801d23d4e89b76e94c1b94b389544168b6cb894f84f8ba62848cf767d72e7f7f4b9d2d7ba07fee33760f79abe5597a51520e292a0cb" }
\ No newline at end of file
+public_key = "ed01208BA62848CF767D72E7F7F4B9D2D7BA07FEE33760F79ABE5597A51520E292A0CB"
```

Comment:
`public_key` is specified in `base.toml`, so here it is redundant for tests. Please revert.